ruby-challenges.rb

```diff
@@ -30,3 +90,42 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+# class Bike 
+#     attr_accessor :model, :wheels, :current_speed
+#     def initialize (model)
+#         @model = model
+#         @wheels = 2
+#         @current_speed = 0
+#     end
+
+#     def bike_info
+#         "The #{@model} has #{@wheels} wheels and is going #{current_speed} mph"
+#     end
+
+#     def pedal_faster(speed_increase)
+#         @current_speed += speed_increase
+#     end
+
+#     def brake(speed_decrease)
+#         @current_speed -= speed_decrease
+
+#         if @current_speed < 0 
+#             @current_speed = 0
+#         end
+        
+
+#     end
+
+
+# end
+
+#  bike_one = Bike.new ('Trek bike')
+
+# p bike_one.bike_info >> 'The Trek bike has 2 wheels and is going 0 mph.'
+
+# p bike_one.pedal_faster(10) >> 10
+# p bike_one.pedal_faster(15) >> 25
+# p bike_one.brake(10) >> "nil"
+
+# I was not able to get my brake method to work properly, what am I missing?
```

Comment:
You were getting `nil` because you are trying to print the return of the `brake` method, but it doesn't have an output defined for if the current_speed is greater than 0. To fix this, you can create an `else` statement with your first line of code, `@current_speed -= speed_decrease` within that `else` statement. However, in your code right now, if you tried to print `bike_info` instead of `pedal_faster` or `brake`, the current speed would display just fine! Nice job!